src/components/EditWalletModal/EditWalletModal-container.tsx

```diff
@@ -26,7 +26,7 @@ const EditWalletModal = ({ id, address, type, description, blockchain, update, e
     const data = {
       wallet_address: address,
       blockchain_id: blockchain,
-      email_id: 'scalac@scalac.io',
+      email_id: email,
```

Comment:
a to nie jest zmienione na useremail czy coś takiego w poprzednim pr była zmnieniona nazwa tego pola ale nie pamiętam na jaką :(

Reply:
Nie widziałam nigdzie przekazywanego tego propsa. Wcześniej dodałam commita, który pobiera ze store'a aktualny email, ale na czas demo został zahardcodowany taki email. Teraz pora już zmienić. Chyba, że mówimy o dwóch różnych rzeczach, co też jest możliwe ;)